zingolib/src/wallet/transaction_records_by_id/trait_inputsource.rs

```diff
@@ -289,60 +289,6 @@ impl InputSource for TransactionRecordsById {
     ) -> Result<Option<zcash_client_backend::wallet::WalletTransparentOutput>, Self::Error> {
         unimplemented!()
     }
-    /// Returns a list of unspent transparent UTXOs that appear in the chain at heights up to and
```

Comment:
I don't see these added back anywhere...does the new librustzcash make them no longer needed?

Reply:
Running against latest dev with the following diff, several tests fail. They're needed on latest dev. If they're not needed with new librustzcash, that's great! 
```diff
git diff
diff --git a/zingolib/src/wallet/tx_map_and_maybe_trees/trait_stub_inputsource.rs b/zingolib/src/wallet/tx_map_and_maybe_trees/trait_stub_inputsource.rs
index f9abbb77..aa90003e 100644
--- a/zingolib/src/wallet/tx_map_and_maybe_trees/trait_stub_inputsource.rs
+++ b/zingolib/src/wallet/tx_map_and_maybe_trees/trait_stub_inputsource.rs
@@ -44,22 +44,4 @@ impl InputSource for TxMapAndMaybeTrees {
             .select_spendable_notes(account, target_value, sources, anchor_height, exclude)
             .map_err(TxMapAndMaybeTreesTraitError::InputSource)
     }
-
-    fn get_unspent_transparent_output(
-        &self,
-        _outpoint: &zcash_primitives::transaction::components::OutPoint,
-    ) -> Result<Option<zcash_client_backend::wallet::WalletTransparentOutput>, Self::Error> {
-        unimplemented!()
-    }
-
-    fn get_unspent_transparent_outputs(
-        &self,
-        address: &zcash_primitives::legacy::TransparentAddress,
-        max_height: zcash_primitives::consensus::BlockHeight,
-        exclude: &[zcash_primitives::transaction::components::OutPoint],
-    ) -> Result<Vec<zcash_client_backend::wallet::WalletTransparentOutput>, Self::Error> {
-        self.transaction_records_by_id
-            .get_unspent_transparent_outputs(address, max_height, exclude)
-            .map_err(TxMapAndMaybeTreesTraitError::InputSource)
-    }
 }
 ```